INSTALL

```diff
@@ -112,16 +112,37 @@ Building the documentation
 0) make sure both PYTHONDIR and PYTHONDIR\Scripts is in your system PATH
    where PYTHONDIR is the directory where you installed Python
 
-1) Install sphinx 
+1) Install sphinx
 2) Open a command prompt
 3) Go to the documentation directory. For example:
- 
+
   cd liam2\doc\usersguide\
 
-3) Launch the build: 
+3) Launch the build:
 
   make html
 
 4) Open the result in your favourite web browser. It is located in:
-  
+
   build/html/index.html
+
+
+Installing on the GNU/linux debian distribution
+-----------------------------------------------
+
+Install cython
+
+  aptitude install cython
+
+Install cx_freeze cx_freeze
+
+  aptitude install mercurial

```

Comment:
I mean: remove it from the install instructions, not from the setup.py where I do need it (but it could be made optional, which would be a good thing).